librocksdb_sys/crocksdb/c.cc

```diff
@@ -1064,6 +1064,17 @@ void crocksdb_write(crocksdb_t* db, const crocksdb_writeoptions_t* options,
   SaveError(errptr, db->rep->Write(options->rep, &batch->rep));
 }
 
+void crocksdb_write_multi_batch(crocksdb_t* db,
+                                const crocksdb_writeoptions_t* options,
+                                crocksdb_writebatch_t** batches,
+                                size_t batch_size, char** errptr) {
+  std::vector<WriteBatch*> ws;
+  for (size_t i = 0; i < batch_size; i++) {
+    ws.push_back(&batches[i]->rep);
+  }
+  SaveError(errptr, db->rep->MultiBatchWrite(options->rep, std::move(ws)));
```

Comment:
Better make `MultiBatchWrite` accepts pointer instead, so that allocation is not needed on C API.

Reply:
Got it, I'll follow it up, thanks!